stacks/spica/src/passport/passport-routing.module.ts

```diff
@@ -50,7 +52,15 @@ const routes: Routes = [
           {path: ":id/edit", component: StrategiesAddComponent, data: {action: "show"}}
         ]
       },
-      {path: "settings", component: IdentitySettingsComponent}
+      {path: "settings", component: IdentitySettingsComponent},
+      {
+        path: "api-key",
+        children: [
+          {path: "", component: ApiKeyIndexComponent},
+          {path: "add", component: ApiKeyAddComponent},
+          {path: ":id/edit", component: ApiKeyAddComponent}
+        ]
+      }
```

Comment:
IdentityGuard, PolicyGuard guards are missing in newly introduced endpoints which will let the user access these pages without required policies.

Reply:
But in this stage, we don't have any service like **passport:api-index** and action like **index,update,show**.